credsweeper/scanner/scan_type/pem_key_pattern.py

```diff
@@ -154,9 +154,11 @@ def sanitize_line(cls, line: str, recurse_level: int = 5) -> str:
 
         # Note that this strip would remove `\n` but not `\\n`
         line = line.strip(string.whitespace)
-        if line.startswith("// "):
-            # assume, the commented line has to be separated from base64 code. Otherwise, it may be a part of PEM.
-            line = line[3:]
+        if line.startswith("//"):
+            # simplify first condition for speed-up to process doxygen style
+            if line.startswith("// ") or line.startswith("/// "):
+                # assume, the commented line has to be separated from base64 code. Otherwise, it may be a part of PEM.
```

Comment:
```suggestion
                # Assume that the commented line is to be separated from base64 code, it may be a part of PEM, otherwise.
```

Reply:
done in https://github.com/Samsung/CredSweeper/pull/477